SQL/versions/32_addDefaultGameTileIconConfig.sql

```diff
@@ -5,6 +5,6 @@ UPDATE jam
 SET jam_default_icon_url = "logo.png";
 
 INSERT INTO config 
-(config_key, config_value, config_category, config_description, config_type, config_options, config_editable, config_required, config_added_to_dictionary)
+(config_lastedited, config_lasteditedby, config_key, config_value, config_category, config_description, config_type, config_options, config_editable, config_required, config_added_to_dictionary)
 VALUES 
-('DEFAULT_GAME_ICON_URL', 'logo.png', 'NEW_JAM_DEFAULTS', 'URL to the default entry icon for entries without a custom icon', 'TEXT', '[]', '1', '1', '1');
\ No newline at end of file
+(Now(), '1', 'DEFAULT_GAME_ICON_URL', 'logo.png', 'NEW_JAM_DEFAULTS', 'URL to the default entry icon for entries without a custom icon', 'TEXT', '[]', '1', '1', '1');
```

Comment:
1 for config_lasteditedby is potentially a valid user id, set to -1 (aka the OVERRIDE_AUTOMATIC_NUM, defined in authentication.php) This will have it attributed to "automatic".